CHANGES_NEXT_RELEASE

```diff
@@ -3,3 +3,4 @@
 - Fix: case-sensitive header duplication (e.g. "Content-Type" and "Content-type") in custom notifications (#2893)
 - Fix: bug in GTE and LTE operations in query filters (q/mq), both for GET operations and subscriptions (#2995)
 - Fix: Wrong "max one service-path allowed for subscriptions" in NGSIv2 subscription operation (#2948)
+- Fix: Bug that may contextBroker to crash when passing inappropriate parameter in query context payload (#3055)
```

Comment:
I think correct issue number is `#3119`.

In addition, given you are going to edit this CNR line, change contextBroker -> ContextBroker, pls.

Reply:
Done. :)

Reply:
this was actually my last changed CNR ;)